webdriver/support/fixtures.py

```diff
@@ -142,3 +144,41 @@ def inner(path, query="", fragment=""):
                                   fragment))
         return rv
     return inner
+
+def create_dialog(session):
+    """Create a dialog (one of "alert", "prompt", or "confirm") and provide a
+    function to validate that the dialog has been "handled" (either accepted or
+    dismissed) by returning some value."""
+
+    def create_dialog(dialog_type, text=None, result_var=None):
```

Comment:
Instead of taking a `result_var` to write a custom global, maybe you could generate a variable name and return it from `create_dialog`?

I think I would prefer an API that looked like this (pseudo-code):

```py
my_dialog = create_dialog(session)("prompt", "What do you like most?")
session.alert.set_text("Cheese!")
assert_dialog_accepted(my_dialog, "Cheese!")
```